config/environments/production.rb

```diff
@@ -70,7 +70,7 @@
   # config.active_support.deprecation = :notify
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
```

Comment:
@whereismyjetpack  is this comment relevant?

Reply:
it is not! mended.

Reply:
thanks!